homeassistant/components/light/__init__.py

```diff
@@ -129,6 +135,22 @@ def turn_on(hass, entity_id=None, transition=None, brightness=None,
     hass.services.call(DOMAIN, SERVICE_TURN_ON, data)
 
 
+# pylint: disable=too-many-arguments
+def set_brightness(hass, entity_id=None, transition=None, brightness=None,
```

Comment:
It should only take arguments that it is using. Please remove the other ones. You can also remove the pylint disable above